### ISSUE TITLE
Accessibility: Signal is literally unusable after recent color change

### ISSUE THREAD

**Issue by lstor** [signalapp/Signal-Android#11389]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Existing issues point out the annoyances, frustrations and lack of logic in the new color design. This issue intends to address the accessibility issue.

After suffering a serious traumatic brain injury (TBI) a handful of years ago, I have had significantly reduced cognitive capacity. Any kind of cognitive load is a strain, and I have a very limited daily "budget". This means I am virtually completely socially isolated, with seeing other people roughly 10 minutes a week on average. Signal is my only lifeline to friends and family. Fortunately, because of the head injury, I have had much sympathy and luck in converting my friends to Signal users, and literally everyone I know except my grandmother is on Signal. Needless to say, this small sliver of "social" contact is paramount to me in an already extremely poor quality of life.

However, after the color changes, I am literally unable to use Signal. One important aspect of my brain injury is that well-trained tasks are easier on my head. Having used Signal since it was renamed Signal, the existing color patterns and color identities were well established. I tried adapting after the change, but my brain working excessively to make sense of the changes wore me out in seconds. Extra cognitive load that is unnoticeable to healthy people is detrimental and very painful to me.

I have tried setting my color to the old contact colors, and I even tried switching my monitors to monochrome, but to no avail. Signal is unusable to me now.

I beg you. I have been completely out of touch from friends and family since this change made its way to my computer, and I'm not sure how longer I can hang on.

*Please* change the colors back, or add an option to do so.

### Steps to reproduce
- Get head injury
- Be isolated from friends and family
- Rely on Signal for communication
- Get completely isolated by Signal color change

**Actual result:** Unable to use Signal due to color changes.
**Expected result:** Can happily use Signal because of old color scheme.

### Device info
**Device:** PC
**Signal version:** Latest

**Comment by mlo-poc** [comment]
There's a real lengthy discussion in the user-forum:

https://community.signalusers.org/t/beta-feedback-for-the-new-color-changes/33048/1133

good to see, you're not alone and **there are solutions** - at least users proposed a bunch.

**Comment by greyson-signal** [comment]
Hey, send me an email at greyson@signal.org and I'll see what I can do to help.

**Comment by Nostrocean** [comment]
They expect people cease complain, when the solution are three lines of code, which would leave both systems active, at the user's choice in the color panel.

https://community.signalusers.org/t/beta-feedback-for-the-new-color-changes/33048/1241

Do not stop complaining. and do not donate. do not recommend donate, explain the reasons... once captured, you don't matter. Business are business.

**Comment by 0mid** [comment]
First, thank you to Signal folks for their world-changing work.
I wanted to echo what others have said: the new chat color change is nonsensical and breaks organization of messages based on the color of OTHER PEOPLE'S texts, i.e., the previous, sensible colors.

Please change the default back to what it used to be, and, if you think the new colors would make sense to some, provide an option for the users to "flip" the colors.

**Comment by klues** [comment]
I really don't like the new way of chat colors anyway, but if you have to do it, please at least make the color of the name the same as the round bubble on the left side. Now it is just really confusing.

![0HRfbmILn5](https://user-images.githubusercontent.com/2537025/123399535-01b6a800-d5a5-11eb-9498-da759f94fdc3.png)

**Comment by greyson-signal** [comment]
Given that this is technically a duplicate of other closed issues, and that I've written many posts about it on [the forum](https://community.signalusers.org/t/beta-feedback-for-the-new-color-changes/33048/1241), I'm going to close this.

But again, given your unique situation, feel free to reach out to greyson@signal.org and I'll see if there's anything I can do.

**Comment by lstor** [comment]
@greyson-signal I sent an email on 24 June (Thursday a week ago) at 09:13 with subject "Unable to use Signal with new color scheme due to TBI"; did you not get it?

**Comment by greyson-signal** [comment]
@lstor Apologies, I did get it, but I somehow missed it. I've responded now.
